datasets/sentinel-products-ca-mirror.yaml

```diff
@@ -0,0 +1,30 @@
+Name: Sentinel Products Canada Mirror
+Description: "The official Government of Canada 🍁 mirror for Sentinel-1,2,3 data under the EU's Copernicus programme focused on Canada-wide coverage. Additional Sentinel-2 and Sentinel-6 products over Canada are retrieved from [EUMETSAT](https://eoportal.eumetsat.int/cas/login)"
+Documentation: https://roda.sentinel-hub.com/sentinel-s1-l1c/GRD/readme.html
+Contact: eodms-sgdot@nrcan-rncan.gc.ca
+ManagedBy: "[Natural Resources Canada](https://www.nrcan.gc.ca/)"
+UpdateFrequency: Sentinel-1 is a near real-time (NRT) dataset retrieved from ESA's [collaborative hubs](https://colhub.copernicus.eu/), not the open hub. Other sensors are updated as quickly as possible based on Canada coverage availability at the source.
+Collabs:
+  ASDI:
+    Tags:
+      - satellite imagery
+Tags:
+  - agriculture
+  - earth observation
+  - satellite imagery
+  - geospatial
+  - sustainability
+  - disaster response
+  - synthetic aperture radar
```

Comment:
feel free to add tags: -stac, -cog

Reply:
ill add stac, but not cog as none are that in fact.

Reply:
done.